.github/workflows/release.yml

```
@@ -0,0 +1,51 @@
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      semver:
+        description: 'The semver to use'
+        required: true
+        default: 'patch'
+        type: choice
+        options:
+          - auto
+          - patch
+          - minor
+          - major
+          - prerelease
+          - prepatch
+          - preminor
+          - premajor
+  pull_request:
+    types: [closed]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      issues: write
+      pull-requests: write
+    steps:
+      - name: Use Node.js 20
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20
+      - uses: nearform-actions/optic-release-automation-action@v4
+        with:
+          commit-message: 'Release {version}'
+          sync-semver-tags: true
+          access: 'public'
+          # This prefix is added before the prerelease number, e.g. `v3.0.0-alpha.0`
+          prerelease-prefix: 'alpha'
+          semver: ${{ github.event.inputs.semver }}
+          # Prereleases are published under the `next` npm dist-tag
+          npm-tag: ${{ startsWith(github.event.inputs.semver, 'pre') && 'alpha' || 'latest' }}
```

This comment doesn't match the behavior. I think we should use the `next` dist-tag.

```suggestion
          npm-tag: ${{ startsWith(github.event.inputs.semver, 'pre') && 'next' || 'latest' }}
```